tests/nat/tools/test_code_execution_sandbox.py

```
@@ -0,0 +1,447 @@
+    response = requests.post(sandbox_config["execute_url"], json=payload)
+
+    # Should return an error status code or error in response
+    assert response.status_code != 200 or "error" in response.json()
+
+
+@pytest.mark.integration
+def test_missing_timeout_field(sandbox_config):
+    """Test request missing the timeout field."""
+    payload = {"generated_code": "print('test')", "language": "python"}
+
+    response = requests.post(sandbox_config["execute_url"], json=payload)
```

Ruff (0.14.3)

Probable use of `requests` call without timeout

(S113)